eva-accession-import-automation/generate_custom_assembly_report.py

```diff
@@ -108,15 +109,14 @@ def update_non_equivalent_genbank_accessions_in_assembly_report(assembly_report_
         logger.info("No entries were found in the assembly report with non-equivalent GenBank accessions")
 
     for accession in refseq_accessions_with_non_equivalent_genbank_accessions:
-        equivalent_genbank_accession = \
-            genbank_equivalents_dataframe[genbank_equivalents_dataframe["rs_acc"] == accession]["gb_acc"]
-        if len(equivalent_genbank_accession) == 0:
-            logger.warn("Could not find equivalent GenBank accession for RefSeq accession " + accession)
-        else:
-            equivalent_genbank_accession = equivalent_genbank_accession.values[0]
+        try:
+            equivalent_genbank_accession = \
+                genbank_equivalents_dataframe.loc[accession]["gb_acc"]
             assembly_report_dataframe.loc[assembly_report_dataframe["RefSeq-Accn"] == accession, "GenBank-Accn"] = \
```

Comment:
The script is failing when the GenBank contig appears more than once in the equivalences file provided by ncbi.

```GCF_000003025.5	GCA_000003025.4	AF034253.1	NC_000845.1```
```GCF_000003025.6	GCA_000003025.6	AF034253.1	NC_000845.1```